requester/error.go

```diff
@@ -11,4 +11,6 @@ var (
 	ErrConnection = errors.New("connection error")
 	// ErrReporting is returned when the Requester fails to send the report.
 	ErrReporting = errors.New("reporting error")
+	// ErrRequestBody is returned when the Requester fails to get the body of the request to copy it
+	ErrRequestBody = errors.New("request body error")
```

Comment:
```suggestion
	ErrCloneBody = errors.New("error cloning the request body")
```

Reply:
Wouldn't it be confusing for a user to get an error _cloning the request body_? 🤔

Reply:
I have let it as it is for the moment, it should be easily modifiable if we want to later!